eng/Version.Details.xml

```diff
@@ -52,7 +52,7 @@
       <Sha>d40c654c274fe4f4afe66328f0599130f3eb2ea6</Sha>
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.runtime" Version="9.0.0-preview.2.24080.1" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.runtime.linux-x64" Version="9.0.0-preview.2.24080.1" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
This is the one drawback to explicitly declaring the intermediates.  They are no longer platform portable.  In practice this is a non-issue because the SB CI legs only run on linux-x64.  @ellahathaway - could you update the SB issue with this callout?